### ISSUE TITLE
Can't add the "Browser Mod" integration in Settings 

### ISSUE THREAD

**Issue by tenkaibr** [thomasloven/hass-browser_mod#592]
I performed all the procedures contained in the instructions but when I go to add the integration, only a window without options appears.

![image](https://github.com/thomasloven/hass-browser_mod/assets/39348260/af7edc19-4c09-41a6-a7f7-4b1a9a1e7611)

![image](https://github.com/thomasloven/hass-browser_mod/assets/39348260/67c766b9-675b-4696-a261-dd588f439d91)

I already cleared the cache and that didn't help either.

HA is up to date: Home Assistant 2023.6.2 Supervisor 2023.06.2 Operating System 10.3 Frontend: 20230608.0 - latest

**Comment by tenkaibr** [comment]
After a lot of digging I discovered that the integration was disabled. When installing the integration, the window was blank and the integration was added to the disable tab. Then I had to reinstall and clear the cache. I updated from version 1.3.1 so here is the log in case anyone has the same error

![image](https://github.com/thomasloven/hass-browser_mod/assets/39348260/5b0d516d-472f-442f-981b-0a76d6c6f2b8)

**Comment by tenkaibr** [comment]
After a lot of digging I discovered that the integration was disabled. When installing the integration, the window was blank and the integration was added to the disable tab. Then I had to reinstall and clear the cache. I updated from version 1.3.1 so here is the log in case anyone has the same error

![image](https://github.com/thomasloven/hass-browser_mod/assets/39348260/5b0d516d-472f-442f-981b-0a76d6c6f2b8)
